### DISCUSSION TITLE
[General] Best way to get OP on OP Sepolia, any DEX deployed with web UI?

### DISCUSSION THREAD

**Discussion by zjmiller** [ethereum-optimism/developers#296] [General]
### Discussion Topic

I'm trying to test some stuff that uses OP.

**Comment by hash1go** [comment]
Hello,

There is no way to acquire OP tokens on OP Sepolia. The OP token contract is pre-deployed and can be checked [here](https://sepolia-optimism.etherscan.io/address/0x4200000000000000000000000000000000000042).
As OP tokens are ERC-20 tokens, if you want to test, please deploy your own token contract.
